### ISSUE TITLE
Linux: Text boxes too small in Audio Device Settings

### ISSUE THREAD

**Issue by SteveDaulton** [audacity/audacity#3383]
### Bug description

Text boxes too small in Audio Device Settings.
See screenshot

### Steps to reproduce

See screenshot

### Expected behavior

The text box to be big enough to display the text.

### Actual behavior

The text box is too small.

### Audacity Version

current master / alpha / nightly build

### Operating system

Linux

### Additional context

Using Xfce with 120 dpi font scaling (required on my laptop so that text is readable).

![Screenshot_2022-08-17_12-33-40](https://user-images.githubusercontent.com/11681097/185109443-c922fd14-9e6d-47a3-8338-0126cea39ff3.png)

**Comment by LWinterberg** [comment]
just like #503, this will get addressed with the Qt version at earliest.

**Comment by SteveDaulton** [comment]
> just like #503, this will get addressed with the Qt version at earliest.

How about if I make a pull request? Just tweaking the minimum width would take 10 seconds of development time. As it is now, it's misleading (it looks like the default latency compensation is -13ms when in fact it is -130ms), and it looks very amateurish.

![Screenshot_2022-08-17_17-33-09](https://user-images.githubusercontent.com/11681097/185194197-17a43041-1b3d-4508-88b4-ff310d75a903.png)

**Comment by LWinterberg** [comment]
> How about if I make a pull request? 

feel free to

**Comment by SteveDaulton** [comment]
> feel free to

Done: https://github.com/audacity/audacity/pull/3398

I've tested on Linux and Windows but not on macOS.
This is the result on Linux:
![Screenshot_2022-08-19_13-03-52](https://user-images.githubusercontent.com/11681097/185615368-4d82deff-1c87-4aee-a971-318dd36841b1.png)

While testing, I noticed another, more serious bug. **_Latency compensation does not work!_**
Regardless of the setting, the offset between the first and subsequent recorded tracks remains the same. I'll log this as a separate issue.
